test/helpers/getPerformanceTests.ts

```diff
@@ -61,7 +61,7 @@ const createNewRunnableTests = (numberOfTest) => {
 		newRunnableTests.add(randomTestName);
 	}
 
-	testFs.writeFileSync(PERFORMANCE_TESTS_FILE, [...newRunnableTests].join('\n'));
+	testFs.writeFileSync(PERFORMANCE_TESTS_FILE, [...Array.from(newRunnableTests)].join('\n'));
```

Comment:
Weird, spreading a Set should work...

In any case, applying the spread on Array.from is unnecessary. Just `Array.from(...).join` has the same effect.

Reply:
This might also be your node version acting up. When I last ran this locally, it all worked

Reply:
Ah, got it. This is the error:

```
> fontoxpath@3.17.0 qt3performance
> ts-node -P test/tsconfig.json -r tsconfig-paths/register test/qt3testsBenchmark.ts


/home/martin/projects/fontoxml/fontoxpath/node_modules/ts-node/src/index.ts:434
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
test/helpers/getPerformanceTests.ts:64:51 - error TS2569: Type 'Set<unknown>' is not an array type or a string type. Use compiler option '--downlevelIteration' to allow iterating of iterators.

64  testFs.writeFileSync(PERFORMANCE_TESTS_FILE, [...newRunnableTests].join('\n'));
                                                     ~~~~~~~~~~~~~~~~

    at createTSError (/home/martin/projects/fontoxml/fontoxpath/node_modules/ts-node/src/index.ts:434:12)
    at reportTSError (/home/martin/projects/fontoxml/fontoxpath/node_modules/ts-node/src/index.ts:438:19)
    at getOutput (/home/martin/projects/fontoxml/fontoxpath/node_modules/ts-node/src/index.ts:578:36)
    at Object.compile (/home/martin/projects/fontoxml/fontoxpath/node_modules/ts-node/src/index.ts:775:32)
    at Module.m._compile (/home/martin/projects/fontoxml/fontoxpath/node_modules/ts-node/src/index.ts:858:43)
    at Module._extensions..js (internal/modules/cjs/loader.js:1272:10)
    at Object.require.extensions.<computed> [as .ts] (/home/martin/projects/fontoxml/fontoxpath/node_modules/ts-node/src/index.ts:861:12)
    at Module.load (internal/modules/cjs/loader.js:1100:32)
    at Function.Module._load (internal/modules/cjs/loader.js:962:14)
    at Module.require (internal/modules/cjs/loader.js:1140:19)
npm ERR! code 1
npm ERR! path /home/martin/projects/fontoxml/fontoxpath
npm ERR! command failed
npm ERR! command sh -c ts-node -P test/tsconfig.json -r tsconfig-paths/register test/qt3testsBenchmark.ts

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/martin/.npm/_logs/2021-03-10T08_36_04_602Z-debug.log
```

Reply:
@bwrrp of course! I removed the spread.

Reply:
@DrRataplan Ah! I think updating the `target` in tsconfig to ES2015 (or above) should allow that to work. For es5 it would have to include a polyfill for iterators, which is why that is disabled by default.

Reply:
Ah, that makes sense. See #299 where I update the compile target to a recent ES